src/main/java/ch/njol/skript/effects/EffFeed.java

```diff
@@ -25,52 +25,55 @@
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
 import org.bukkit.entity.Player;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Feed")
 @Description("Feeds the specified players.")
-@Examples({"feed all players", "feed the player by 5 beefs"})
+@Examples({
+	"feed all players",
+	"feed the player by 5 beefs"
```

Comment:
Let's advertise the correct usage :D
```suggestion
	"feed the player by 5 hunger"
```